### PR TITLE
Add something

### PR DESCRIPTION
**Pull request by lliioollcn**
# Title
添加一些消息屏蔽

## 介绍 / Description
添加屏蔽所有消息
添加屏蔽所有卡片消息
添加屏蔽所有红包消息

脚本因一些问题不能正常解析，在以后修复

## 更改内容 / Types of Changes

- 新功能 New Feature
- 增强优化 Enhancement/optimization 

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  #486

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Review by singleNeuron — CHANGES_REQUESTED** [review]
请参照这两个类处理用户消息https://github.com/ferredoxin/QNotified/blob/master/app/src/main/java/me/singleneuron/hook/decorator/SimpleCheckIn.kt
https://github.com/ferredoxin/QNotified/blob/master/app/src/main/java/me/singleneuron/qn_kernel/dispacher/ItemBuilderFactoryHook.kt

**Review by qwq233 — CHANGES_REQUESTED** [review]
https://github.com/ferredoxin/QNotified/blob/master/CONTRIBUTING.md
commit message要求简洁明了无中文  如果你要提交3个功能则需要分3个commit

**Review by ketal178 — COMMENTED** [review]
Commit Message乱写
一个Commit只干一件事

File: app/src/main/java/cn/lliiooll/utils/MsgRecord.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
+package cn.lliiooll.utils;
```
Comment (singleNeuron) [inline app/src/main/java/cn/lliiooll/utils/MsgRecord.java:1]: https://github.com/ferredoxin/QNotified/blob/master/app/src/main/java/me/singleneuron/data/MsgRecordData.kt
不要重复造轮子

**Review by singleNeuron — CHANGES_REQUESTED** [review]

**Comment by qwq233** [comment]
image
仔细读下

File: app/src/main/java/nil/nadph/qnotified/script/QNScriptManager.java [hunk 2]
```diff
@@ -21,19 +21,17 @@
 import android.widget.CompoundButton;
```
Comment (singleNeuron) [inline app/src/main/java/nil/nadph/qnotified/script/QNScriptManager.java:21]: 此部分更改是否与拦截消息功能有关？为什么整个PR里都没有提这部分修改？

**Review by singleNeuron — CHANGES_REQUESTED** [review]

**Pull request closed without merging**